meson.build

```diff
@@ -102,6 +102,7 @@ localstatedir = join_paths('/', get_option('localstatedir'))
 rootbindir = join_paths(rootprefixdir, 'bin')
 rootsbindir = join_paths(rootprefixdir, split_bin ? 'sbin' : 'bin')
 rootlibexecdir = join_paths(rootprefixdir, 'lib/systemd')
+sysrpath = rootlibexecdir + ':' + get_option('sysrpath')
```

Comment:
I think this doesn't really work well in case sysrpath is empty...

(Though, empty is probably a valid value, meaning "this directory", as in ".", though considering you can achieve that using an explicit "." that's probably better.)

I'd at least handle the case where this is unset correctly.

I think your use case is too specific for an upstream patch... But I think this patch doesn't really make things too bad, so I'm not opposed to merging it in. (Not too excited about it either.)